spec/models/lettings_log_spec.rb

```diff
@@ -486,18 +486,56 @@ def previous_postcode_fields
       end
 
       describe "when changing a log's scheme and hence calling reset_scheme_location!" do
-        let(:scheme) { FactoryBot.create(:scheme) }
-        let(:invalid_location_1) { FactoryBot.create(:location, scheme:, startdate: Time.zone.today + 3.weeks) }
-        let(:valid_location) { FactoryBot.create(:location, scheme:, startdate: Time.zone.yesterday) }
-        let(:invalid_location_2) { FactoryBot.create(:location, scheme:, startdate: Time.zone.today + 3.weeks) }
+        before do
+          Timecop.return
```

Comment:
This file still has some wide ranging dates-in-the-far-past stuff going on - I've left it alone to not interfere with anything currently going on elsewhere to fix that, but we should make sure it gets sorted.